dockerfiles/Dockerfile.webpack

```diff
@@ -0,0 +1,5 @@
+FROM node:16
```

Comment:
Might be worth noting why we're still in 16 here.

Reply:
Ah, I may be confused with `website` maybe 🤔 . I updated this image to use 22.11 and it worked 💯